src/Theme/utils.ts

```diff
@@ -123,3 +123,9 @@ export enum TEXT_VARIANT {
   NEGATIVE = 'negative',
   BASE = 'base',
 }
+
+export enum ARIA_LIVE_VARIANT {
+  OFF = 'off',
+  ASSERTIVE = 'assertive',
+  POLITE = 'polite',
+}
```

Comment:
Can we also provide documentation on what the different values here signify? i.e "Use `ASSERTIVE` in this case. Use `POLITE` in this one."